package.json

```diff
@@ -0,0 +1,101 @@
+{
+  "name": "task",
+  "version": "0.1.0",
+  "private": true,
+  "description": "",
+  "keywords": [
+    "react",
+    "webpack",
+    "typescript",
+    "babel",
+    "sass",
+    "hmr",
+    "starter",
+    "boilerplate"
+  ],
+  "author": "Kanstantsin Kulahin",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": ""
+  },
+  "homepage": ".",
+  "scripts": {
+    "build": "npm run clean-dist && webpack --progress --config=configs/webpack/prod.js --env NODE_ENV=production",
+    "clean-dist": "rimraf dist/*",
+    "lint": "tsc --noEmit && eslint src/**/*.{js,ts,tsx} --no-error-on-unmatched-pattern",
+    "stylelint": "npx stylelint src/**/*.s{c,a}ss",
+    "lint:fix": "tsc --noEmit && eslint --fix src/**/*.{js,ts,tsx} --no-error-on-unmatched-pattern",
+    "stylelint:fix": "npx stylelint --fix src/**/*.s{c,a}ss",
+    "lint:all": "npm run lint:fix && npm run stylelint:fix",
+    "start:dev": "webpack serve --config=configs/webpack/dev.js --env NODE_ENV=development --env BUILD_TYPE=local",
+    "start:prod": "npm run build && npm start",
+    "start": "node express.js"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.13.14",
```

Comment:
It is better not to make dependencies dynamic.

Reply:
Fixed